eng/pipelines/templates/steps/analyze.yml

```diff
@@ -48,6 +48,10 @@ steps:
       node eng/tools/rush-runner.js lint "${{parameters.ServiceDirectory}}"
     displayName: "Build ESLint Plugin and Lint Libraries"
 
+  - pwsh: |
```

Comment:
What does an error message look like from this script? It would be nice if the error message told someone exactly what command to run to fix the formatting as opposed to trying to fix individual formatting issues.

Reply:
This PR already fails because of the new command and here is one failure: https://dev.azure.com/azure-sdk/public/_build/results?buildId=754165&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=a880e989-7d1a-5c96-a41f-d540b383cc43&l=53. The command does not try to fix it but it shows a list of files that need to be formatted. The PR author will need to run `rushx format` locally to format if their text editor did not do that already so I do not think there is a need to print a special message.

Reply:
I leave that up to you guys but I would say that someone new coming in trying read that log and figure out how to fix it might be a little difficult. I find that when the error messages are really clear and tell people exactly what to do they cut down on support questions and empower people to easily fix their own issues.

Reply:
Fair enough. I will consider using a different command other than `rushx check-format`.

Reply:
You could also just do a check in the inline pwsh script for the errors and given a message there.

Reply:
I added the check in the js script here: https://github.com/Azure/azure-sdk-for-js/pull/13983/commits/b05ec077910dd1fba55a43288defb67555a89e7e.